registry/storage/driver/base/base.go

```diff
@@ -104,8 +120,17 @@ func (base *Base) GetContent(ctx context.Context, path string) ([]byte, error) {
 
 // PutContent wraps PutContent of underlying storage driver.
 func (base *Base) PutContent(ctx context.Context, path string, content []byte) error {
-	ctx, done := dcontext.WithTrace(ctx)
-	defer done("%s.PutContent(%q)", base.Name(), path)
+	attrs := []attribute.KeyValue{
+		attribute.String(tracing.AttributePrefix+"driver.name", base.Name()),
+		attribute.String(tracing.AttributePrefix+"path", path),
+		attribute.Int("content.length", len(content)),
```

Comment:
I don't see an attribute with that name in the OTEL semantic conventions. Did you mean to prefix it with `tracing.AttributePrefix`?